src/Containers/Microsoft.NET.Build.Containers/ImageBuilder.cs

```diff
@@ -32,15 +31,15 @@ internal sealed class ImageBuilder
     public ImageConfig BaseImageConfig => _baseImageConfig;
 
     /// <summary>
-    /// MediaType of the output manifest.
+    /// MediaType of the output manifest. By default, this will be the same as the base images' manifest.
```

Comment:
typo image's ?

Reply:
There's a quirk of English grammar with regards to possessive case I think that makes my form correct, but it would be clearer to remove the possessive entirely:

```suggestion
    /// MediaType of the output manifest. By default, this will be the same as the base image manifest.
```